sql/updates/latest-dev.sql

```diff
@@ -170,8 +170,14 @@ DECLARE
   chunk regclass;
 BEGIN
   FOR chunk IN
-  SELECT pg_catalog.format('%I.%I', schema_name, table_name)::regclass
-    FROM _timescaledb_catalog.chunk WHERE dropped = false
+    SELECT pg_catalog.format('%I.%I', schema_name, table_name)::regclass
```

Comment:
According to @feikesteenbergen suggestion, perform the alter statement only when it is needed.